src/components/version-notification.riot

```diff
@@ -3,7 +3,7 @@
   <material-popup opened="{ state.open }" onClick="{ onClose }">
     <div class="material-popup-title">Check for updates</div>
     <div class="material-popup-content">
-      <p>The version <b>{ state.tag_name }</b> of Docker Regisrty UI now available.</p>
+      <p>The version <b>{ state.tag_name }</b> of Docker registry UI now available.</p>
```

Comment:
This is the name of project, keep the upercase, revert this change

Reply:
Thanks. Done